server/scheduler.go

```diff
@@ -35,3 +35,27 @@ func scheduleLeader(cluster *clusterInfo, s Selector) (*regionInfo, *storeInfo,
 
 	return region, source, target
 }
+
+func scheduleStorage(cluster *clusterInfo, s Selector) (*regionInfo, *storeInfo, *storeInfo) {
```

Comment:
The function name indicates what kind of resource it schedules, and this function schedules for storage.